index.html

```diff
@@ -54,6 +55,174 @@ <h2 class="connect">LET'S CONNECT</h2>
+				</div>
+				 <p class="day">
+					A daily selection of privately personalized reads;
+					no accounts or sign-ups required.
+				 </p>
+				<ul class="langs">
+					<li class="btn1">
+						html
+					</li>
+					<li class="btn2">
+						css
+					</li>
+					<li class="btn3">
+						javaScript
+					</li>
+				</ul>
+				<button class="submit" type="submit">See Project</button>
+			</div>
+			<div class="works2">
+				<img class="snap2" src="images/SnapshootPortfolios.svg" alt="">
+				<h2 class="tonic2">Multi-Post Stories</h2>
+				<div class="grid-container2">
+				  <div class="grid-twelve">
+				    <p>CANOPY</p>
+				</div>
+				  <div class="grid-twentytwo">
+					<p>.</p>
+				 </div>
+				  <div class="grid-thirtytwo">
+					<p>Back End Dev</p>
+				 </div>
+				  <div class="grid-fortytwo">
+					<p>.</p>
+				 </div>
+				  <div class="grid-fiftytwo">
+					<p>2015</p>
+				 </div>
+				</div>
+				<p class="day2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="langs2">
+					<li class="btn12">html</li>
+					<li class="btn22">css</li>
+					<li class="btn32">javaScript</li>
+				</ul>
+				<button class="btn42" type="submit">See Project</button>
+			</div>
+			<div class="works3">
+				<img class="snap3" src="images/SnapshootPortfolio.svg" alt="">
+				<h2 class="tonic3">Tonic</h2>
+				<div class="grid-container3">
+				  <div class="grid-13">
+					<p>CANOPY</p>
+				  </div>
+				  <div class="grid-23">
+					<p>.</p>
+				  </div>
+				  <div class="grid-33">
+					<p>Back End Dev</p>
+				  </div>
+				  <div class="grid-43">
+					<p>.</p>
+				  </div>
+				  <div class="grid-53">
+					<p>2015</p>
+				  </div>
+				</div>
+
+				<p class="day3">
+					A daily selection of privately personalized reads; 
+					no accounts or sign-ups required.
+				</p>
+				<ul class="langs3">
+					<li class="btn13">html</li>
+					<li class="btn23">css</li>
+					<li class="btn33">javaScript</li>
+				</ul>
+				<button class="btn43" type="submit">See Project</button>
+			</div>
+			<div class="works4">
+				<img class="snap4" src="images/snapshot4.svg" alt="">
+				<h2 class="tonic4">Multi-Post Stories</h2>
+				<div class="grid-container4">
+				  <div class="grid-14"><p>CANOPY</p></div>
+				  <div class="grid-24"><p>.</p></div>
+				  <div class="grid-34"><p>Back End Dev</p></div>
+				  <div class="grid-44"><p>.</p></div>
+				  <div class="grid-54"><p>2015</p></div>
+				</div>
+				<p class="day4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<ul class="langs4">
+					<li class="btn14">html</li>
+					<li class="btn24">css</li>
+					<li class="btn34">javaScript</li>
+				</ul>
+				<button class="btn44" type="submit">See Project</button>
+			</div>
+		</section>
+	<div class="div-class">
+		<section class="about-myself">
+      		<div class="desk-about">
+        		<h1 class="aboutmy">About <br> Myself</h1>
```

Comment:
Kindly change the font size and font color of this heading and text to match the Figma Design 🤗

Your Design:
![image](https://user-images.githubusercontent.com/73714615/185757289-83c556b5-107e-4459-a0c4-c39c8a78ca25.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/185757302-7b4a32c4-edaa-4b5f-b2e7-eb025ae12beb.png)